Agreements/Airport/PB_Mor.txt

```diff
@@ -1 +1,3 @@
-COPX:EGPB:*:*:*:*:PBAPP:Moray:11000:*:^Y905
\ No newline at end of file
+COPX:EGPB:*:*:*:*:PBAPP:Moray:12000:*:^Y905
+COPX:EGPM:*:*:*:*:PBAPP:Moray:12000:*:^Y905
+COPX:EGET:*:*:*:*:PBAPP:Moray:12000:*:^Y905
```

Comment:
I've done some digging in the SRD, which I hadn't managed to get round to before creating the issue, and found that there are a number of standard routes out of here that don't go via the Y905! Therefore, I think it would be best to define these agreements using a FIXAFTER of ADN. I don't think there are any routes that go southbound via the Y905 that _don't_ route via ADN? 

(The reason why it doesn't make sense to use MOCHA or something closer, however, is that there are some routes that route straight to BEKET/PETOX which I think we would still want this agreement to apply to in a VATSIM context).